modules/uhttp/filters.go

```diff
@@ -70,7 +70,10 @@ func tracingServerFilter(host service.Host) FilterFunc {
 		}
 		span := opentracing.GlobalTracer().StartSpan(operationName, ext.RPCServerOption(spanCtx))
 		ext.HTTPUrl.Set(span, r.URL.String())
-		defer span.Finish()
+		defer func() {
+			span.Finish()
+			fxctx.Logger().Info("Span finished")
```

Comment:
That information is already added to the logger by the time defer is called. We add trace id and span id to the logger.